package.json

```diff
@@ -143,8 +143,8 @@
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
     "react-native-onyx": "1.0.118",
-    "react-native-pager-view": "^6.2.0",
-    "react-native-pdf": "^6.7.3",
+    "react-native-pager-view": "6.2.2",
+    "react-native-pdf": "^6.7.4",
```

Comment:
Whoops. This has caused https://github.com/Expensify/App/issues/34510 which @situchan previously solved by downgrading version.

@situchan Can you downgrade this again?

Reply:
Weird. Seems like merge conflict.
Gitlens also showing old [PR](https://github.com/Expensify/App/pull/33561)

Reply:
Then revert it again to fix this 😄